src/ioctl/linux.rs

```diff
@@ -12,7 +12,7 @@ pub(super) const fn compose_opcode(
 ) -> RawOpcode {
     macro_rules! shift_and_mask {
         ($val:expr, $shift:expr, $mask:expr) => {{
-            ($val << $shift) & $mask
```

Comment:
Should we rename the macro to `mask_and_shift`?

Reply:
Good idea, renamed.